Tests/test_imagemath.py

```diff
@@ -5,11 +5,11 @@
 from PIL import Image, ImageMath
 
 
-def pixel(im):
-    if hasattr(im, "im"):
-        return f"{im.mode} {repr(im.getpixel((0, 0)))}"
+def pixel(im: Image.Image | int) -> str | int:
     if isinstance(im, int):
         return int(im)  # hack to deal with booleans
+    else:
+        return f"{im.mode} {repr(im.getpixel((0, 0)))}"
```

Comment:
We can remove this `else`:
```suggestion

    return f"{im.mode} {repr(im.getpixel((0, 0)))}"
```